packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -5340,17 +5340,30 @@ There are four choices for the `connection` property.
  - `ON_NETWORK` - Displays VPN connection information only when a browser's client IP matches the configured Pubic Gateway IPs. The notification appears before the end user can access the app.
  - `OFF_NETWORK` - Displays VPN connection information only when the browser's client IP doesn't match the configured Pubic Gateway IPs. The notification appears before the end user can access the app.
 
-#### Attribute Statements Object
+### Attribute Statements Object
 
 Specifies (optional) attribute statements for a SAML application.
 
 | Property   | Description                                                                                  | DataType    | Nullable |
 | ---------- | -------------------------------------------------------------------------------------------- | ----------- | -------- |
 | name       | The reference name of the attribute statement                                                | String      | FALSE    |
 | namespace  | The name format of the attribute                                                             | String      | FALSE    |
-| values     | The value of the attribute; Supports [Okta EL](/docs/reference/okta-expression-language/)      | String      | FALSE    |
+| values     | The value of the attribute; Supports [Okta EL](/docs/reference/okta-expression-language/)    | String      | FALSE    |
+| type       | The type of attribute statements object                                                      | `GENERIC`   | FALSE    |
 
-##### Supported Namespaces
+#### Group Attribute Statements Object
+
+Group Attribute Statements can be used in place of Attribute Statements if your org supports a large number of groups and you wish to filter them into a single SAML assertion.
```

Comment:
we capitalize Org and avoid the use of "wish" :)

Group Attribute Statements can be used in place of Attribute Statements if your Org supports a large number of groups, and you want to filter them into a single SAML assertion.